dockerfiles/is/Dockerfile

```diff
@@ -59,9 +59,10 @@ WORKDIR ${USER_HOME}
 # set environment variables
 ENV JAVA_HOME=${JAVA_HOME} \
     PATH=$JAVA_HOME/bin:$PATH \
-    WSO2_SERVER_HOME=${WSO2_SERVER_HOME}
+    WORKDIR=${USER_HOME} \
+    CLUSTERING_ENABLED=false
 
 # expose ports
-EXPOSE 4000 9763 9443 
+EXPOSE 4000 9763 9443
 
-ENTRYPOINT ${WSO2_SERVER_HOME}/bin/wso2server.sh
+ENTRYPOINT ${WORKDIR}/init.sh
```

Comment:
@chirangaalwis The pull request looks good! I think it might be more readable if we can rename init.sh file to entrypoint.sh. WDYT?

Reply:
+1 for the suggestion.